cmd/root/root.go

```diff
@@ -109,6 +114,38 @@ func rootPersistentPreRunEFunc(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
+
+	// Handle chunk size flags (deprecation and overwriting where needed)
+	//
+	// Expected Behavior for chunk size flags:
+	// * If either cli option is set, use that value
+	// * If both are set, emit an error
+	// * If neither are set, use ENV values
+	// ** If PGET_CHUNK_SIZE is set, use that value
+	// ** If PGET_CHUNK_SIZE is not set, use PGET_MINIMUM_CHUNK_SIZE if set
+	//    NOTE: PGET_MINIMUM_CHUNK_SIZE value is just set over the key for PGET_CHUNK_SIZE
+	//    Warning message will be emitted
+	// ** If both PGET_CHUNK_SIZE and PGET_MINIMUM_CHUNK_SIZE are set, use PGET_CHUNK_SIZE
+	//    Warning message will be emitted
+	// * If neither are set, use the default value
+
+	changedMin := cmd.PersistentFlags().Changed(config.OptMinimumChunkSize)
+	changedChunk := cmd.PersistentFlags().Changed(config.OptChunkSize)
+	if changedMin && changedChunk {
+		return fmt.Errorf("--minimum-chunk-size and --chunk-size cannot be used at the same time, use --chunk-size instead")
+	} else if !(changedMin && changedChunk) {
```

Comment:
```suggestion
	} else {
```
in this house, we accept the [principal of the excluded middle](https://en.wikipedia.org/wiki/Law_of_excluded_middle), so this condition is unnecessary.